framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/TemplateProjectBuilder.cs

```diff
@@ -111,6 +112,11 @@ public async Task<ProjectBuildResult> BuildAsync(ProjectBuildArgs args)
                 args
             );
 
+            if (context.Template is AppTemplateBase appTemplateBase)
+            {
+                appTemplateBase.HasDbMigrations = SemanticVersion.Parse(templateFile.Version) < new SemanticVersion(4, 4, 0);
```

Comment:
@maliming This will be False for 4.4.0 preview versions. ( v4.4.0 > v4.4.0-preview.1)

Reply:
I think this will be fine.

`appTemplateBase.HasDbMigrations = SemanticVersion.Parse(templateFile.Version) < new SemanticVersion(4, 3, 99);`

Reply:
https://github.com/abpframework/abp/pull/9194/commits/e8d287183eef35523e4def20f064ec13d267a4a2